File: python_backend/ranges/specialMoves/enPassant/get_t_sqr.py

from ranges.specialMoves.enPassant.en_passants import en_passants

def get_t_sqr(color, align_key):
    """use alignment key to get square that pawn moves to capture by en-passant"""
    return en_passants[color][align_key]

if __name__ == "__main__":
    print(get_t_sqr('B', 'b4a4'))
    print(get_t_sqr('W', 'c5d5'))

File: node.js/ranges/specialMoves/enPassant/getTSqr.js
const { rfToXy } = require("../../../coordType/crdCnvrt");
const  enPassants  = require("./enPassants");


function getTSqr(color, alignKey) {
    /*use alignment key to get square that pawn moves to capture by en-passant**/
    var tSqr = enPassants[color][alignKey]
    return tSqr;
}

module.exports = getTSqr;

if (require.main === module) {
    console.log(getTSqr('B', 'b4a4'))
    console.log(getTSqr('W', 'c5d5'))
}